arctic/store/_pandas_ndarray_store.py

```diff
@@ -220,6 +213,8 @@ class PandasPanelStore(PandasDataFrameStore):
 
     @staticmethod
     def can_write_type(data):
+        # late import, Panel is removed in pandas 1
+        from pandas import Panel
```

Comment:
Will the import happen every time can_write_type gets called?

Reply:
a module is only loaded once by the interpreter. all subsequent imports are returning the already-loaded module, it's very quick.